synapse/tools/cryo/cat.py

```diff
@@ -1,14 +1,18 @@
 import sys
+import json
 import pprint
 import argparse
+import logging
 
 import synapse.common as s_common
 import synapse.cryotank as s_cryotank
 
 import synapse.lib.output as s_output
 import synapse.lib.msgpack as s_msgpack
 
-def main(argv, outp=s_output.stdout):
+logger = logging.getLogger(__name__)
+
+def main(argv, outp=s_output.stdout, errfd=sys.stderr):
```

Comment:
``errfd`` is unused here?

Reply:
We should add a test for it if we're going to keep it

Reply:
removed

Reply:
:+1: